README.md

```diff
@@ -69,20 +67,29 @@ pip install -r requirements/base.txt
 npm install
 ```
 
-### Install testing dependencies
-The integration tests require [PhantomJS](http://phantomjs.org/) to be installed globally.
+### Install testing system dependencies
+We have a suite of end-to-end integration tests. We use [ChromeDriver](http://chromedriver.chromium.org/) to drive headless Chrome or Chromuim. This step isn't required to get a dev install running but will be required to run the full test suite.
+
+On ubuntu, run
+
+```
+sudo apt-get install chromium-browser chromium-chromedriver
+```
+
+to install the dependencies. The chromedriver executable needs to be in `PATH`, so either add `/usr/lib/chromium-browser/chromedriver` to `PATH` or create a symlink e.g:
 
-On Mac OSX, this can be installed by running
 ```
-brew install phantomjs
+sudo ln -s /usr/lib/chromium-browser/chromedriver /usr/local/bin/chromedriver
 ```
 
-On Ubunutu, run:
+On Mac OSX, run:
 ```
-sudo npm install -g phantomjs-prebuilt
+brew install brew-cask
+brew cask install google-chrome
+brew cask install chromedriver
```

Comment:
@symroe - I've attempted to fill in something sensible here, but obviously I haven't tested this. Can you let me know if this actually works/gets you to a stage where the tests will run. If this is wrong or we need extra steps, let me know and I'll correct.

Reply:
First line needs to be 

```bash
brew tap homebrew/cask
```

`brew cask install google-chrome` isn't needed if the user already has Google Chrome installed using the conventional (non-brew) method of installing apps.